src/App.tsx

```diff
@@ -1,12 +1,62 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import { makeStyles } from "@mui/styles";
+import { Button } from "@mui/material";
+import { Box } from "@mui/system";
 
 
-function App() {
+const useStyles = makeStyles({
+  container: {
+    width: '90%',
+    margin: '50px auto',
+    textAlign: 'center'
+  },
+  h1: {
+    color: 'burlywood',
+    fontSize: '3rem',
+  },
+  board: {
+    maxWidth: '450px',
+    margin: '25px auto',
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  square: {
+    width: '150px',
+    height: '150px',
+    border: '2px solid burlywood',
+    cursor: 'pointer'
+  },
+});
+
+const App = () => {
+  const [boardTable, setBoardTable] = useState(Array(9).fill(null));
+  const [turn, setTurn] = useState(false);
+
+  const classes = useStyles();
+
   return (
-    <h1>Tic Tac Toe with Typescript</h1>
+    <div className={classes.container}>
```

Comment:
Material UI offers a [Container ](https://mui.com/components/container/)component; however this is the right way to implement a container if you want to avoid mui container for some reason.

Reply:
Done, I changed the div for the Container component and gave it the styles